deps/config/migrations/000001_create_tables.up.sql

```diff
@@ -0,0 +1,51 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS secrets (
+    id VARCHAR (50) NOT NULL,
+    version VARCHAR (50) NOT NULL,
+    store_id VARCHAR (50) NOT NULL,
+    disabled BOOLEAN default false,
+    tags JSONB,
+    created_at TIMESTAMPTZ DEFAULT (now() at time zone 'utc') NOT NULL,
+    updated_at TIMESTAMPTZ DEFAULT (now() at time zone 'utc') NOT NULL,
+    deleted_at TIMESTAMPTZ,
+    PRIMARY KEY (id, version, store_id)
```

Comment:
Compose PRIMARY KEY are very expensive in DBs, therefore I would rather create a autoincrement PK and then an UNIQUE (id, version, store_id)

Reply:
done